c2corg_ui/static/js/viewdetails.js

```diff
@@ -450,6 +577,26 @@ app.ViewDetailsController.prototype.openEmbeddedImage = function(imgUrl, imgId)
   pswp.init();
 };
 
+/**
+ * @param {string} filename
+ * @param {string} suffix
+ * @return {string}
+ * @export
+ */
+app.ViewDetailsController.prototype.getBandeau = function(filename, suffix) {
```

Comment:
Really used? I see no other occurence of it?!
Looks like a duplicate of the next function.